javaparser-core/src/main/java/com/github/javaparser/ast/NodeList.java

```diff
@@ -53,11 +53,7 @@
     private List<AstObserver> observers = new ArrayList<>();
 
     public NodeList() {
-        this((Node) null);
-    }
-
-    public NodeList(Node parent) {
-        setParentNode(parent);
+        parentNode = null;
```

Comment:
How is this related?

Reply:
Nothing happened here but reformatting of IDE generated code.

Reply:
I wanted to create a nodelist with a single item, but very confusingly, if you have only one item it becomes the parent instead of an item?!? This constructor wasn't used anywhere so I removed it. Deprecation would mean that it got in the way for longer. Hopefully nobody depends on this behaviour.